src/jsonrpc-virtual-machine.cpp

```diff
@@ -188,250 +258,261 @@ jsonrpc_mg_mgr::jsonrpc_mg_mgr(std::string remote_address) {
     m_address.clear();
 }
 
-bool jsonrpc_mg_mgr::is_shutdown(void) const {
+bool jsonrpc_mgr::is_shutdown(void) const {
     return m_address.empty();
 }
 
-jsonrpc_virtual_machine::jsonrpc_virtual_machine(jsonrpc_mg_mgr_ptr mgr) : m_mgr(std::move(mgr)) {}
+jsonrpc_virtual_machine::jsonrpc_virtual_machine(jsonrpc_mgr_ptr mgr) : m_mgr(std::move(mgr)) {}
 
-jsonrpc_virtual_machine::jsonrpc_virtual_machine(jsonrpc_mg_mgr_ptr mgr, const std::string &directory,
+jsonrpc_virtual_machine::jsonrpc_virtual_machine(jsonrpc_mgr_ptr mgr, const std::string &directory,
     const machine_runtime_config &runtime) :
     m_mgr(std::move(mgr)) {
     bool result = false;
-    jsonrpc_request(m_mgr->get_mgr(), m_mgr->get_remote_address(), "machine.machine.directory",
+    jsonrpc_request(m_mgr->get_stream(), m_mgr->get_remote_address(), "machine.machine.directory",
         std::tie(directory, runtime), result);
 }
 
-jsonrpc_virtual_machine::jsonrpc_virtual_machine(jsonrpc_mg_mgr_ptr mgr, const machine_config &config,
+jsonrpc_virtual_machine::jsonrpc_virtual_machine(jsonrpc_mgr_ptr mgr, const machine_config &config,
     const machine_runtime_config &runtime) :
     m_mgr(std::move(mgr)) {
     bool result = false;
-    jsonrpc_request(m_mgr->get_mgr(), m_mgr->get_remote_address(), "machine.machine.config", std::tie(config, runtime),
-        result);
+    jsonrpc_request(m_mgr->get_stream(), m_mgr->get_remote_address(), "machine.machine.config",
+        std::tie(config, runtime), result);
 }
 
 jsonrpc_virtual_machine::~jsonrpc_virtual_machine(void) = default;
 
 machine_config jsonrpc_virtual_machine::do_get_initial_config(void) const {
     machine_config result;
-    jsonrpc_request(m_mgr->get_mgr(), m_mgr->get_remote_address(), "machine.get_initial_config", std::tie(), result);
+    jsonrpc_request(m_mgr->get_stream(), m_mgr->get_remote_address(), "machine.get_initial_config", std::tie(), result);
     return result;
 }
 
-machine_config jsonrpc_virtual_machine::get_default_config(const jsonrpc_mg_mgr_ptr &mgr) {
+machine_config jsonrpc_virtual_machine::get_default_config(const jsonrpc_mgr_ptr &mgr) {
     machine_config result;
-    jsonrpc_request(mgr->get_mgr(), mgr->get_remote_address(), "machine.get_default_config", std::tie(), result);
+    jsonrpc_request(mgr->get_stream(), mgr->get_remote_address(), "machine.get_default_config", std::tie(), result);
     return result;
 }
 
-semantic_version jsonrpc_virtual_machine::get_version(const jsonrpc_mg_mgr_ptr &mgr) {
+semantic_version jsonrpc_virtual_machine::get_version(const jsonrpc_mgr_ptr &mgr) {
     semantic_version result;
-    jsonrpc_request(mgr->get_mgr(), mgr->get_remote_address(), "get_version", std::tie(), result);
+    jsonrpc_request(mgr->get_stream(), mgr->get_remote_address(), "get_version", std::tie(), result);
     return result;
 }
 
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wunused-parameter"
```

Comment:
I'm almost running `git blame` on this to see who got tired of `(void)`. Was it me? 🤨